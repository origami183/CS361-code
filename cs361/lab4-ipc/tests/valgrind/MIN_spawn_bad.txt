==1844363== Memcheck, a memory error detector
==1844363== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1844363== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1844363== Command: ../ipc -s ../data/f5.txt
==1844363== 
CKSUM(../data/f5.txt) = ''
==1844363== 
==1844363== HEAP SUMMARY:
==1844363==     in use at exit: 0 bytes in 0 blocks
==1844363==   total heap usage: 3 allocs, 3 frees, 8,993 bytes allocated
==1844363== 
==1844363== All heap blocks were freed -- no leaks are possible
==1844363== 
==1844363== For lists of detected and suppressed errors, rerun with: -s
==1844363== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
