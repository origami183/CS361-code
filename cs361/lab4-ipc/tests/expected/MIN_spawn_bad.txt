CKSUM(../data/f5.txt) = ''
